applications/launchpad/gui-react/src/App.tsx

```diff
@@ -25,7 +25,7 @@ const App = () => {
   const themeConfig = useAppSelector(selectThemeConfig)
   const dispatch = useAppDispatch()
 
-  const [onboarding, setOnboarding] = useState(false)
+  const [onboarding, setOnboarding] = useState(true)
```

Comment:
Maybe it should stay `false` because there is no exit button on the onboarding screen

Reply:
Updated.